README.Rmd

```diff
@@ -63,13 +63,39 @@ matched <- match_name(loanbook_demo, ald_demo) %>%
   prioritize()
 ```
 
+### Add Scenario Targets
 * Use `target_sda()` to calculate SDA targets of CO2 emissions.
 
 ```{r}
-target_sda(matched, ald_demo, co2_intensity_scenario_demo)
+target_sda(matched, 
+           ald = ald_demo, 
+           co2_intensity_scenario = co2_intensity_scenario_demo)
 ```
 
-* Use `join_ald_scenario()` to join the matched dataset to the relevant 
+* Use `target_market_share` to calculate market-share scenario targets at the portfolio level:
+
+```{r}
+target_market_share(matched, 
+                    ald = ald_demo, 
+                    scenario = scenario_demo_2020, 
+                    region_isos = region_isos_demo)
+```
+
+* ... or at the company level:
+
+```{r}
+target_market_share(matched, 
+                    ald = ald_demo, 
+                    scenario = scenario_demo_2020, 
+                    region_isos = region_isos_demo,
+                    by_company = TRUE)
+```
+
+### Utility Functions
+While most of the joining and calculations occur under the hood, there are also 
+some utility functions that may be of interest to some analysts:
```

Comment:
I generally prefer a more personal style. I write with someone in mind, such as Sasha -- an analyst at Bank of America. Then "some analysts" becomes "you".

Here the actor is implicitly "the package", or "the code", "we (the developers)" or similar. I would do it explicit.

Also, sentences are generally clearer and active when you map actions to verbs. The first verb here is "occur" and the action is burried in so called nominalizations: Joining and calculations. 

My revision:

While the package joins and calculates [what?] under the hood, it {the package} also includes utility functions you may use:

Maybe this is not exactly what you meant but shows how I aim to review. What most strongly influenced my writing is the "[Readers expectations approach](https://georgegopen.com/reader-expectation-approach)"

The [paper](https://georgegopen.com/wp-content/uploads/2019/12/The-Science-of-Scientific-Writing-1.pdf) that presented the approach includes this summary that I like a lot:

![image](https://user-images.githubusercontent.com/5856545/84555502-f4ed4c00-ace2-11ea-9f34-8674897bd1c9.png)

If we both write using the same systematic approach, our documentation will be more consistent. If you follow some other approach let me know and we can discuss.

Reply:
I gave it another shot, to be more explicit about what i was trying to say, let me know how you like it!